src/main/java/seedu/address/model/person/Person.java

```
@@ -82,7 +83,7 @@ public Address getAddress() {
 
     @Override
     public Set<Tag> getTags() {
```

Need to specify unmodifiable otherwise it will throw `UnsupportedOperationException` if developer tries to modify it (according to java docs).